src/main/java/FilesCreatorAndWriter.java

```diff
@@ -0,0 +1,24 @@
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+
+public class FilesCreatorAndWriter {
+    public static void main(String[] args) throws IOException {
+        try {
+            File file = new File("data.txt");
+            if (!file.exists()) {
+                file.createNewFile();
+            }
+            String pathToFile = "C:\\Users\\Jonatan\\IdeaProjects\\data.txt\\data.txt";
```

Comment:
Użyj ścieżki relatywnej, a nie absolutnej (relative path).
Aktualnie jakbym ja u siebie uruchomił ten program to on nie zadziała, ponieważ mogę nie mieć folderu:

```
C:\\Users\\Jonatan\\IdeaProjects\\
```